tools/bindgen-patches/0001-change-c_char-to-c_uchar-in-ifrn_name.patch

```diff
@@ -0,0 +1,26 @@
+From 8faa5d38ebe9b6bd27fda01dd8f54d3d91583166 Mon Sep 17 00:00:00 2001
```

Comment:
hmm this is a `git format-patch` patch... I'm not sure it is 100% compatible with `patch`. Could we use `git  am` below to apply this patch?

Reply:
mh, `git am` actually fails with 

```
Applying: change c_char to c_uchar in ifrn_name
error: src/vmm/src/devices/virtio/net/gen/iff.rs: does not match index
Patch failed at 0001 change c_char to c_uchar in ifrn_name
hint: Use 'git am --show-current-patch=diff' to see the failed patch
hint: When you have resolved this problem, run "git am --continue".
hint: If you prefer to skip this patch, run "git am --skip" instead.
hint: To restore the original branch and stop patching, run "git am --abort".
hint: Disable this message with "git config advice.mergeConflict false"
```
so I think just having `git diff` output instead of `git format-patch` might work better here :thinking: